src/System.Net.Http/ref/project.json

```diff
@@ -3,14 +3,15 @@
     "System.Runtime": "4.0.0",
     "System.Threading.Tasks": "4.0.0",
     "System.IO": "4.0.0",
-    "System.Net.Primitives": "4.0.0",
+    "System.Net.Primitives": "4.0.10",
+    "System.Security.Cryptography.X509Certificates": "4.1.0-rc3-24008-00",
```

Comment:
Is System.Net.Http part of "standard library"?  I'm not sure I understand what that is and what contracts are part of "standard library". But System.Net.Http needs to have Crypto moving forward.